variables.tf

```diff
@@ -12,3 +12,14 @@ variable "topic_name" {
   description = "The name of the SNS topic."
   type        = string
 }
+
+# ------------------------------------------------------------------------------
+# OPTIONAL PARAMETERS
+#
+# These parameters have reasonable defaults.
+# ------------------------------------------------------------------------------
+variable "topic_access_policy" {
+  default     = null
+  description = "The fully-formed JSOM IAM access policy to apply to the SNS topic."
+  type        = string
```

Comment:
Are we able to use `jsondecode()` and `can()` here to do some validation to at least ensure that valid JSON is being provided?

Reply:
Good idea!  Please see commit 181658a.

Reply:
Excellent, thanks!